docs/_data/cta.yml

```diff
@@ -28,17 +28,17 @@ operations:
       icon: <i class="fa fa-arrow-down"></i>
 
 local:
-  title: Run Conjur
+  title: Run Your Own Conjur Server
   text: "Running your own Conjur server gives you total control over the software and doesn't require you to connect to the Internet."
   links:
     self-host:
       title: Install Conjur
-      content: <a href="/get-started/install-conjur.html" class="conjur-cta-primary">Get Started</a>
+      content: <a href="/get-started/install-conjur.html" class="conjur-cta-primary">Install Conjur</a>
 
 hosted:
-  title: Demo Conjur
+  title: Evaluate Conjur in the Cloud
```

Comment:
This is really the easiest option for most people, as it requires relatively little memory and download time. Should we list this first, to the left of "Run Your Own Conjur Server"?

Reply:
I had it that way initially, but it was decided to flip them because we wanted to try and direct users to install vs evaluation

Reply:
Oh that's intentional, good then. I thought we wanted people to use the cloud eval.